lib/api/uikit/class-beans-uikit.php

```diff
@@ -76,22 +107,14 @@ public function compile() {
 		 *
 		 * @param array $components An array of UIkit script compiler arguments.
 		 */
-		$scripts_args = apply_filters(
+		$args = apply_filters(
 			'beans_uikit_euqueued_scripts_args',
```

Comment:
Can we correct this typo? 
Deprecate `beans_uikit_euqueued_scripts_args` and use `beans_uikit_enqueued_scripts_args` instead?

Reply:
I thought about that.  But the filter may be used elsewhere by plugins or themes.  It could cause headaches for people.  So I opted to just leave it.

I know...it goes against my better nature too.

Reply:
Oh well... it's being moved into a plugin anyway 🙈 🙊

Reply:
Exactly.  We can deal with it once we separate it out.